src/trusted/service_runtime/lind_syscalls.c

```diff
@@ -616,9 +534,9 @@ StubType stubs[] = {
         {0}, /* 53 */
         {0}, /* 54 */
         {0}, /* 55 */
-        {LindEpollCreatePreprocess, LindEpollCreatePostprocess, 0}, /* 56 epoll_create */
-        {LindEpollCtlPreprocess, 0, 0}, /* 57 epoll_ctl */
-        {LindEpollWaitPreprocess, LindEpollWaitPostprocess, 0}, /* 58 epoll_wait */
+        {0}, /* 56 epoll_create */
+        {0}, /* 57 epoll_ctl */
+        {0}, /* 58 epoll_wait */
```

Comment:
It also might be good to remove these comments, I've been trying to do so to keep track of what we've gotten rid of (calls that don't need any preprocessing etc. will just have their name commented and still have the array entry be {0}